Description: Disable the Android logger since the android platform isn't available
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/isoparser/src/main/java/com/googlecode/mp4parser/util/AndroidLogger.java
+++ b/isoparser/src/main/java/com/googlecode/mp4parser/util/AndroidLogger.java
@@ -1,6 +1,6 @@
 package com.googlecode.mp4parser.util;
 
-import android.util.Log;
+//import android.util.Log;
 
 /**
  * Logs on Android.
@@ -15,16 +15,16 @@
 
     @Override
     public void logDebug(String message) {
-        Log.d(TAG, name + ":" + message);
+        //Log.d(TAG, name + ":" + message);
     }
 
     @Override
     public void logWarn(String message) {
-        Log.w(TAG, name + ":" + message);
+        //Log.w(TAG, name + ":" + message);
     }
 
     @Override
     public void logError(String message) {
-        Log.e(TAG, name + ":" + message);
+        //Log.e(TAG, name + ":" + message);
     }
 }
